pyrate/timeseries.py

```diff
@@ -317,7 +315,7 @@ def _solve_ts_lap(nvelpar, velflag, ifgv, mat_b, smorder, smfactor, sel, vcmt):
     tsvel = np.empty(nvelpar, dtype=float32) * np.nan
     tsvel[~np.isclose(velflag, 0.0, atol=1e-8)] = x[:nvelleft]
 
-    # TODO: implement uncertainty estimates (tserror) like in Matlab Pirate code
+    # TODO: implement uncertainty estimates (tserror)
```

Comment:
without the reference to matlab this becomes really ambiguous - are you sure we should be nuking every mention of the word "matlab"? Some of these comments being deleted could be useful

Reply:
I thought the same thing initailly! Just knowing that a function is a translation of a funtion from Pirate dose not tell me what the funtion is ment to do. I made a note in ticket to pull comments from Matlab implementation if available.